tests/registry/extraDevfileEntries.yaml

```diff
@@ -39,7 +39,7 @@ samples:
   - name: java-springboot-basic
     displayName: Basic Spring Boot
     description: A simple Hello World Java Spring Boot application using Maven
-    icon: https://spring.io/images/projects/spring-edf462fec682b9d48cf628eaf9e19521.svg
+    icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
```

Comment:
can we replace with this icon? https://raw.githubusercontent.com/devfile-samples/devfile-stack-icons/main/spring.svg.  This is what the actual sample has